nibabel/streamlines/header.py

```diff
@@ -0,0 +1,19 @@
+
+class Field:
```

Comment:
You wanted to avoid typos in the nibabel code?   Or for other people using the code?    Do you want to restrict the fields to just these fields?  In that case you could just have a class with `__slots__`, or a numpy structured array, which would do the type checking and field name checking automatically.